internal/assets/contents/executors/executor.bat

```diff
@@ -12,4 +12,4 @@ set {{$K}}={{$V}}
 {{- end}}
 {{- end}}
 
-"{{.stateExec}}" "{{.stateSock}}" "{{.target}}" "{{.projectDir}}" %*
+"{{.stateExec}}" "{{.stateSock}}" "{{.targetFile}}" "{{.nameSpace}}" "{{.commitID}}" "{{.headless}}" %*
```

Comment:
Note you could just pass the dimensions json blob here instead. That's one argument, and can be passed directly to `resolver.RuntimeUsage()`.

Reply:
I'll surely keep an open mind to that if more data needs to pass, but, for now, I'd like to keep it just space separated instead of having the executor marshal anything.